src/projects/republic/config/englishRepublicLabels.ts

```diff
@@ -45,5 +45,7 @@ export const englishRepublicLabels = Object.assign({}, englishLabels, {
   ORG: "organisation",
   PER: "person",
 
+  page: "Page",
```

Comment:
Misschien kunnen we de labels eens recht trekken qua casing (all caps, capitalized, lowercase, camel case, helaas nog geen kebab case), maar dat is buiten de scope van dit PR.